### PR TITLE
Add the current maintainers team

### PR DESCRIPTION
**Pull request by tmattio**
cc @avsm

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,6 +38,17 @@ make start
...
+## Maintainers
+
+The OCaml.org maintainers team is composed of the following community members:
+
+- Thibaut Mattio, Lead Maintainer (Tarides)
+- Christine Rose, Technical Writer (Tarides)
+- Cuihtlauac Alvarado, Maintainer (Tarides)
+- Sabine Schmaltz, Maintainer (Tarides)
+
+We're always looking for new maintainers, if you're interested in helping us make OCaml.org the best resource to learn OCaml and discover the ecosystem, [reach out to us](mailto:thibaut@tarides.com)!
```
Comment (christinerose) [inline README.md]:
```suggestion
We're always looking for new maintainers! If you're interested in helping us make OCaml.org the best resource to learn OCaml and discover the ecosystem, [reach out to us](mailto:thibaut@tarides.com)!
```

**Review by christinerose — APPROVED** [review]
I love it! Thanks for including me! :) 
Just one grammar suggestion from me here.

**Comment by avsm** [comment]
Thanks! Could this list please include everyone with write access to this repository? If they're not a maintainer any more, I'll remove that write access for security reasons:

In addition to the list in this PR, I see
- @jonludlam
- @MirzaBabarBaig
- @patricoferris 
- @pitag-ha 
- @samoht 
- me
- @TheLortex 
- @asaadmahmood 
...with write access to the repository.

File: README.md [hunk 2]
```diff
@@ -38,6 +38,20 @@ make start
 
 See our [contributing guide](./CONTRIBUTING.md) for more detailed instructions.
 
+## Maintainers
+
+The OCaml.org maintainers team is composed of the following community members:
+
+- Anil Madhavapeddy ([@avsm](https://github.com/avsm)), Owner (Cambridge University)
```
Comment (avsm) [inline README.md]:
```suggestion
- Anil Madhavapeddy ([@avsm](https://github.com/avsm)), Owner (University of Cambridge)
```

**Comment by avsm** [comment]
This looks like a good start. I would like to reconcile the remaining contributors with write access with this list -- I'm happy for them to be maintainers, but if they're not active then we can shift them down to read. This is just for good security practises to the main website -- we can reinstate them any time they want to contribute again!

**Comment by tmattio** [comment]
Thanks @avsm! The list in the PR should be the complete list of current maintainers.

Just to be clear: this is the list of people who are actively maintaining the site and should have write access to the ocaml/ocaml.org repository (i.e. who can merge PRs), as per the governance page definition of "maintainer" (https://ocaml.org/governance).

The list of people who contribute to the website, in general, is much larger, and in particular includes the infrastructure team, the maintainers of the package documentation pipeline (@jonludlam  and @TheLortex  in particular), and the many community contributors. Hopefully, the [acknowledgement section](https://github.com/ocaml/ocaml.org#acknowlegement) of the README gives proper credit to everyone who contributed or still contribute to the website, please let me know if notice that we forgot someone!

**Comment by tmattio** [comment]
Merging now - as @avsm said, if you previously maintained the website and would like to contribute again, you can reach out to me and we can reinstate the access!

**Pull request merged**